lib/vagrant.rb

```diff
@@ -139,10 +139,14 @@ def self.configure(version, &block)
     Config.run(version, &block)
   end
 
-  # This checks if a plugin with the given name is installed. This can
-  # be used from the Vagrantfile to easily branch based on plugin
-  # availability.
+  # This checks if a plugin with the given name is available (installed
+  # and enabled). This can be used from the Vagrantfile to easily branch
+  # based on plugin availability.
   def self.has_plugin?(name, version=nil)
+    if !Vagrant.plugins_enabled?
```

Comment:
Oops, you're right, I was doing python stuff right before this and forgot about one liners.